fdbserver/DataDistribution.actor.cpp

```diff
@@ -1559,47 +1623,31 @@ ACTOR Future<Void> bulkLoadJobMarkEmptyRangeComplete(Reference<DataDistributor>
 	return Void();
 }
 
-ACTOR Future<std::unordered_map<Key, BulkLoadManifest>> fetchBulkLoadTaskManifests(
-    std::string jobRoot,
+ACTOR Future<std::unordered_map<Key, BulkLoadJobFileManifestEntry>> fetchBulkLoadTaskManifestEntries(
```

Comment:
This returns BulkLoadJobFileManifestEntry but its called fetchBulkLoadTaskManifestEntries ?